docs/current_docs/integrations/github-actions.mdx

```diff
@@ -18,12 +28,47 @@ When running a CI pipeline with Dagger using a standard GitHub Actions runner, t
 1. The Dagger CLI executes the specified sub-command and sends telemetry to Dagger Cloud if the `DAGGER_CLOUD_TOKEN` environment variable is set.
 1. The workflow completes with success or failure. Logs appear in GitHub as usual.
 
+</TabItem>
+<TabItem value="Depot">
+
+When running a CI pipeline on a Dagger Powered Depot GitHub Actions Runner, the general workflow looks like this:
+
+1. GitHub emits a repository event which triggers a workflow
+1. Depot receives this event, processes it, and runs one or more jobs in the workflow, each running in an isolated Depot GitHub Actions Runner.
```

Comment:
```suggestion
1. Depot receives this event, processes it, and launches an isolated Depot GitHub Actions Runner.
```

GitHub sends webhook events for each job in a workflow. A workflow can have multiple jobs, but a runner is only ever allowed to work on one job.